root/etc/e-smith/events/actions/nethserver-nextcloud-conf

```diff
@@ -35,9 +35,13 @@ else
     if [ -f /usr/share/nextcloud/.htaccess.rpmsave ]; then
         rm -f /usr/share/nextcloud/.htaccess.rpmsave
     fi
+    if [ -f /usr/share/nextcloud/.htaccess.rpmnew ]; then
+        rm -f /usr/share/nextcloud/.htaccess.rpmnew
+    fi
     OCC "maintenance:mode --on"
     OCC "upgrade"
     OCC "maintenance:mode --off"
+    OCC "integrity:check-core"
```

Comment:
What is the purpose of this command? Does it reset all integrity warnings?

Reply:
No, it does not any reset.
It checks the integrity of the code. Using code signing it ensures that nobody [alter the code](https://docs.nextcloud.com/server/12/admin_manual/issues/code_signing.html#why-did-nextcloud-add-code-signing).
It can generate various [errors/warnings messages](https://docs.nextcloud.com/server/12/admin_manual/issues/code_signing.html#fixing-invalid-code-integrity-messages), so for example also new added files generate a warning. This is the reason to remove .rpmnew.